analyzer/windows/modules/packages/ie.py

```diff
@@ -92,6 +134,14 @@ class IE(Package):
                 # "Internet Explorer - Security Warning"
                 # "The publisher could not be verified."
                 "CheckExeSignatures": "no",
+                "RunInvalidSignatures": 1
+            },
+        ],
+        [
+            HKEY_CURRENT_USER,
+            "Software\\Microsoft\\Windows\\CurrentVersion\\Internet Settings\\Zones\\0",
```

Comment:
@doomedraven no ... this is realted to ActiveX ref: https://support.microsoft.com/en-us/kb/182569